index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
This attribute is required for modern webpages. Maybe it should be updated once the page has rendered with a certain language